### DISCUSSION TITLE
My Device's Serial Number is 2390350525. Version5 Does Not Support. Can someone tell me how to use Version4? I cannot read from the data logger.

### DISCUSSION THREAD

**Discussion by basersoft** [community/community#102128] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My Device's Serial Number is 2390350525. Version5 Does Not Support. Can someone tell me how to use Version4? I cannot read from the data logger.

**Comment by queenofcorgis** [comment]
Hi @basersoft ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your device. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by basersoft** [reply]
Thanks
